<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewDialog</class>
 <widget class="QDialog" name="PreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="mediaControls" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="ClickableGraphic" name="skipBack">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="ClickableGraphic" name="playPause">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="ClickableGraphic" name="stop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="ClickableGraphic" name="skipForward">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="exitPreview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="popupMode">
    <enum>QToolButton::MenuButtonPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="arrowType">
    <enum>Qt::DownArrow</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="scene">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>70</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableGraphic</class>
   <extends>QGraphicsView</extends>
   <header>clickablegraphic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
